scripts/main.py

```diff
@@ -18,7 +18,6 @@
 import logging
 from prompt import get_prompt
 
-cfg = Config()
```

Comment:
This is wrong (probably from a merge). Isn't cfg used below on line 26?

Reply:
Correct, should be added back in.